cmake/development.cmake

```diff
@@ -0,0 +1,63 @@
+message(STATUS "Local development is available via docker compose")
+set(GO_BIN_DIR ${CMAKE_BINARY_DIR}/go/bin)
+file(MAKE_DIRECTORY ${GO_BIN_DIR})
+
+# mkcert for local development
+set(MKCERT_EXECUTABLE ${GO_BIN_DIR}/mkcert)
+set(MKCERT_VERSION latest)
+add_custom_command(
+  OUTPUT ${MKCERT_EXECUTABLE}
+  BYPRODUCTS ${MKCERT_EXECUTABLE}
+  COMMAND GOBIN=${GO_BIN_DIR} ${GO_EXECUTABLE} install filippo.io/mkcert@${MKCERT_VERSION}
+  COMMENT "Setting up mkcert"
+)
+
+# hostess for local development
+set(HOSTESS_EXECUTABLE ${GO_BIN_DIR}/hostess)
+set(HOSTESS_VERSION latest)
+add_custom_command(
+  OUTPUT ${HOSTESS_EXECUTABLE}
+  BYPRODUCTS ${HOSTESS_EXECUTABLE}
+  COMMAND GOBIN=${GO_BIN_DIR} ${GO_EXECUTABLE} install github.com/cbednarski/hostess@${HOSTESS_VERSION}
+  COMMENT "Setting up hostess"
+)
+
+set(LOCAL_DOMAIN "monetr.local")
+set(LOCAL_CERTIFICATE_DIR ${CMAKE_BINARY_DIR}/certificates/${LOCAL_DOMAIN})
+set(LOCAL_CERTIFICATE_KEY ${LOCAL_CERTIFICATE_DIR}/key.pem)
+set(LOCAL_CERTIFICATE_CERT ${LOCAL_CERTIFICATE_DIR}/cert.pem)
+file(MAKE_DIRECTORY ${LOCAL_CERTIFICATE_DIR})
+add_custom_command(
+  OUTPUT ${LOCAL_CERTIFICATE_KEY} ${LOCAL_CERTIFICATE_CERT}
+  BYPRODUCTS ${LOCAL_CERTIFICATE_KEY} ${LOCAL_CERTIFICATE_CERT}
+  # TODO Make this work on windows.
+  COMMAND sudo ${MKCERT_EXECUTABLE} -install
```

Comment:
`sudo` does not work on windows. Should look into:

https://superuser.com/questions/42537/is-there-any-sudo-command-for-windows